bossbot/db.py

```diff
@@ -27,11 +27,38 @@ def __init__(
             cursor.execute(GR_SETUP_TABLE)
             cursor.execute(GR_OAUTH_TABLE)
             cursor.execute(LASTFM_USER_TABLE)
+            cursor.execute(KARMA_TRACKER_TABLE)
             self.conn.commit()
 
     def cursor(self, *args, **kwargs):
         return self.conn.cursor(*args, **kwargs)
 
+class KarmaDB(BossDB):
+    def set_user_karma(self, user_id, karma):
```

Comment:
seems like we could also implement increment/decrement functions here to have a slightly better interface. not a big deal, though

Reply:
ack

Reply:
i added increment, but punted on decrement since i wanna add decrement alongside with up/down tracking